app/lib/local_links_manager/import/local_custodian_codes_importer.rb

```diff
@@ -16,7 +16,7 @@ def import_from_csv(csv_file)
             local_authority = LocalAuthority.find_by!(slug: slug)
             local_authority.update!(local_custodian_code: code)
           rescue ActiveRecord::RecordNotFound
-            Rails.logger.info("Not found #{row['Authority']}, not updated")
+            Rails.logger.info("Not found #{row[AUTHORITY_HEADER_INDEX]}, not updated")
```

Comment:
Could even be this (more readable):

```suggestion
            Rails.logger.info("Not found #{slug}, not updated")
```